README.md

```diff
@@ -36,3 +36,45 @@ git fetch
 For further information or help, don't hesitate to get in touch on our [Slack organisation](https://nf-co.re/join/slack) (a tool for instant messaging).
 
 [^1]: From [stackoverflow](https://stackoverflow.com/a/60846265/11502856)
+
+## How data has been acquired
+### For GLIMPSE
+#### Initial data
+Beware can be long to download especially 'NA12878.final.cram'
+```
+wget -c http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/CCDG_14151_B01_GRM_WGS_2020-08-05_chr21.filtered.shapeit2-duohmm-phased.vcf.gz{,.tbi}
+wget -c http://hgdownload.soe.ucsc.edu/goldenPath/hg38/chromosomes/chr21.fa.gz hs38DH.chr21.fa.gz
+wget -c ftp://ftp.sra.ebi.ac.uk/vol1/run/ERR323/ERR3239334/NA12878.final.cram
+wget https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/genome/chr21/germlineresources/gnomAD.r2.1.1.vcf.gz -O gnomAD.r2.1.1.chr21.vcf.gz
+```
+
+#### Preparation of the different panel files
+```
+. get_panel_s.sh
```

Comment:
It would be nice to have a 1-2 sentence short description of what each script is doing